.github/workflows/ci.yml

```diff
@@ -142,6 +142,14 @@ jobs:
         path: build
         key: ${{ runner.os }}-build-${{ github.sha }}
         restore-keys: ${{ runner.os }}-build-
+        
+    - name: Download boost
+      run: |
+        $url = "https://sourceforge.net/projects/boost/files/boost/1.74.0/boost_1_74_0.tar.gz/download"
+        (New-Object System.Net.WebClient).DownloadFile($url, "$env:TEMP\boost.tar.gz")
+        7z.exe x "$env:TEMP\boost.tar.gz" -o"$env:TEMP\boostArchive" -y | Out-Null
+        7z.exe x "$env:TEMP\boostArchive" -o"$env:TEMP\boost" -y | Out-Null
+        Push-Location -Path "$env:TEMP\boost"
```

Comment:
I think this command is useless here, I think it just prepares the installation (that we are not doing).

We can change if it works in anoter PR to not delay more the blocking

Reply:
probably :D
I just copy pasted...

Yes I would merge this PR asap, could you please open a follow up PR with this change (and probably more improvements)?
Thx